kie-server-parent/kie-server-tests/kie-server-integ-tests-drools/src/test/java/org/kie/server/integrationtests/drools/pmml/ApplyScorecardModelIntegrationTest.java

```diff
@@ -51,22 +53,38 @@
     private static final String CONTAINER_ID = "scorecard";
 
     private static final long EXTENDED_TIMEOUT = 300000L;
+    private static ClassLoader classLoader;
 
     @BeforeClass
     public static void buildAndDeployArtifacts() {
         KieServerDeployer.buildAndDeployCommonMavenParent();
         KieServerDeployer.buildAndDeployMavenProjectFromResource("/kjars-sources/scorecard");
+        KieMavenRepository kmp = KieMavenRepository.getKieMavenRepository();
+        File artifact = kmp.resolveArtifact(releaseId).getFile();
```

Comment:
Why is this classsloader magic needed here?

Reply:
This is necessary so that the classes generated from the PMML, and are located in the kjar, are available to the test/application.

Reply:
Are customers supposed to do this classloader configuration to be able to use PMML with XStream?